.github/workflows/upload_alerts.yml

```diff
@@ -0,0 +1,38 @@
+# upload alerts every 15 minutes
+
+name: Upload Alerts to AWS/Rockset
+
+on:
+  schedule:
+    - cron: '*/15 * * * *'
+
+jobs:
+  upload-alerts:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
IIRC, `fetch-depth: 0` gets all the history (expensive checkout) https://github.com/actions/checkout.  I don't think it's needed here.  You can just leave `fetch-depth` out and it will use the default depth of 1 (cheap checkout)